Sito/carrello.php

```diff
@@ -0,0 +1,135 @@
+<?php
```

Comment:
Ho provato intanto a fare il test se entro senza avere niente in carello:
Da dei warning sopra e poi il testo di carrello vuoto se possibile gli metterei un margine a sinistra altrimenti lascia così
![image](https://user-images.githubusercontent.com/56229661/72457977-00862680-37c8-11ea-9def-2838346bbcc2.png)